libs/ledger-live-common/src/families/elrond/bridge.integration.test.ts

```diff
@@ -97,6 +99,25 @@ const elrond: CurrenciesData<Transaction> = {
             warnings: {},
           },
         },
+        {
+          name: "Send max",
+          transaction: fromTransactionRaw({
+            family: "elrond",
+            recipient:
+              "erd1frj909pfums4m8aza596595l9pl56crwdj077vs2aqcw6ynl28wsfkw9rd",
+            useAllAmount: true,
+            amount: "0",
+            mode: "send",
+            fees: null,
+          }),
+          expectedStatus: (account, _, status) => {
+            return {
+              amount: account.balance.minus(status.estimatedFees),
```

Comment:
if balance work we can keep as it is. But better use spendableBalance than balance,

some crypto have a rules that we need to keep some amount of balance to keep the account alive. That's why we have spendable balance to know exactly how much balance we can spend

Reply:
Alright, tbh I used the calculation from the elrond getTransactionStatus: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/src/families/elrond/js-getTransactionStatus.ts#L48

Reply:
hmmm we should be careful with this one because Elrond is getting rework.
But let's keep it as it is and during the merge of the full rework we will let them update this part 👍